frontend-main/dist/index.html

```diff
@@ -1,20 +1,17 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
+  <head>
+    <meta charset="UTF-8" />
+    <script defer src="main.js"></script>
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+    </style>
```

Comment:
今更で申し訳ないんですが、cssは外部ファイルにしたいです(html内ではstylelintが効かないため)
例えば
```suggestion
    <link rel="main.css">
```
など

Reply:
わかりました！今のところCSSは使ってないのですがとりあえずtest.cssにリンクを繋げました